athina/steps/llm.py

```
@@ -268,3 +268,77 @@ def execute(self, input_data: dict, **kwargs) -> str:
+                model=self.model,
+                **self.model_options.model_dump(),
+                **(self.tool_config.model_dump() if self.tool_config else {}),
+                **({"response_format": self.response_format}),
+                **(
+                    kwargs.get("search_domain_filter", {})
+                    if isinstance(kwargs.get("search_domain_filter"), dict)
+                    else {}
+                ),
+            )
+            llmresponse = llm_service_response["value"]
+            output_type = kwargs.get("output_type", None)
```

Ruff (0.8.2)

Use `kwargs.get("output_type")` instead of `kwargs.get("output_type", None)`

Replace `kwargs.get("output_type", None)` with `kwargs.get("output_type")`

(SIM910)